CHANGELOG.md

```diff
@@ -3,19 +3,19 @@ aws-parallelcluster-cookbook CHANGELOG
 
 This file is used to list changes made in each version of the AWS ParallelCluster cookbook.
 
-2.x.x
+2.11.0
 -----
 
 **ENHANCEMENTS**
 - Add support for Ubuntu 20.04.
 - Add support for using FSx Lustre in subnet with no internet access.
 - Add support for Centos 7 ARM.
-- Make sure slurmd service is not enabled before finalize stage, which will prevent user from unintentionally making compute node available in post-install process.
+- Make sure slurmd service is only enabled after post-install process, which will prevent user from unintentionally making compute node available during post-install process.
```

Comment:
Make sure slurmd service is only enabled after post-install process, which will prevent node to become available during post-install process.